### PR TITLE
FW-2483 Allow users to share their photos [Customer Requirement]

### PR DESCRIPTION
**Pull request by Prochnost55**
### What do you want to achieve?
- create a method to enable user to capture images using mobile's camera (on mobile devices)

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [x] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

- open a web page in mobile on which the widget is loaded. 
- the initialization script should have `capturePhoto: true` in kommunicateSettings

https://user-images.githubusercontent.com/19753380/126484890-b20666ca-ad9f-4dfd-a5c8-8e30c5634f00.mp4

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/css/app/mck-sidebox-1.0.css [hunk 1]
```diff
@@ -1818,6 +1818,12 @@ ul.mck-intent-menu>li:hover {
 #mck-btn-attach-box.mck-text-box-panel {
 	width: auto !important;
 }
+#mck-attachImg-box{
```
Comment (devashishmamgain) [inline webplugin/css/app/mck-sidebox-1.0.css:1821]: use lowercase for css

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 2]
```diff
@@ -3740,6 +3750,12 @@ var userOverride = {
                           'n-vis',
                           'vis'
                       );
+                
+                !IS_CAPTURE_PHOTO && kommunicateCommons.modifyClassList(
+                    { id: ['mck-imgFile-up']},
```
Comment (devashishmamgain) [inline webplugin/js/app/mck-sidebox-1.0.js:3755]: either use camelCase or use -

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

File: webplugin/css/app/mck-sidebox-1.0.css (hunk 1 above)
Comment (Prochnost55) [reply webplugin/css/app/mck-sidebox-1.0.css]: done

File: webplugin/js/app/mck-sidebox-1.0.js (hunk 2 above)
Comment (Prochnost55) [reply webplugin/js/app/mck-sidebox-1.0.js]: done

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 3]
```diff
@@ -14083,7 +14107,13 @@ var userOverride = {
                     );
                     $mck_file_input.trigger('click');
                 });
-
+                $mck_img_upload.on('click', function (e) {
+                    e.preventDefault();
+                    // kmWidgetEvents.eventTracking(
```
Comment (Ojashwa) [inline webplugin/js/app/mck-sidebox-1.0.js:14112]: @devashishmamgain 
- Shall we add event tracking for this like we have added for the othe buttons?

**Review by devashishmamgain — APPROVED** [review]

File: webplugin/js/app/mck-sidebox-1.0.js (hunk 3 above)
Comment (devashishmamgain) [reply webplugin/js/app/mck-sidebox-1.0.js:14112]: yes

**Review by devashishmamgain — APPROVED** [review]

**Review by Ojashwa — APPROVED** [review]

**Review by ChannuBS — APPROVED** [review]

**Pull request merged**